services/bundle_analysis/notify/contexts/__init__.py

```diff
@@ -188,14 +184,13 @@ def load_user_config(self) -> "NotificationContextBuilder":
         This allows all notifiers to access configuration for any notifier and already have the defaults
         """
         comment_config: bool | dict = self.current_yaml.read_yaml_field("comment")
-        if not comment_config:
-            required_changes = False
+        required_changes: bool | Literal["bundle_increase"]
+        if isinstance(comment_config, bool):
+            required_changes = comment_config
             required_changes_threshold = BundleThreshold("absolute", 0)
         else:
-            required_changes: bool | Literal["bundle_increase"] = comment_config.get(
-                "require_bundle_changes", False
-            )
-            required_changes_threshold: int | float = comment_config.get(
+            required_changes = comment_config.get("require_bundle_changes", False)
```

Comment:
nit: can we have `required_changes` not return a boolean? I think returning `None` is a bit more intuitive, especially since the other type is a string literal.